### DISCUSSION TITLE
`export const` vs. `export default` in ES6 둘의 차이는 무엇인가요?

### DISCUSSION THREAD

**Discussion by hochan222** [transcendence42/javascript-archive#13] [Q&A] [answered]
**기본 내보내기 (export default):** 사용자가 이름을 지정할 수 있음

**명명 된 내보내기 (export):** 이름을 지정해서 내보내기 때문에 사용자가 이름을 지정할 수 없음, 사용자가 이름을 지정해주기 위한 추가 문법 필요

- https://stackoverflow.com/questions/33611812/export-const-vs-export-default-in-es6
- https://ko.javascript.info/import-export

**Comment by yhshin0** [comment, marked as answer]
`export const`는 read-only로만 내보낸다는 의미를 갖고 있습니다.

그리고 `export default`는 해당 모듈에서 하나의 개체만 내보낸다는 의미를 갖고 있습니다.

따라서, `export const`를 import 하는 경우에는 {}를 사용해야 하고, `export deafult`를 import 하는 경우에는 {}를 사용하지 않아도 됩니다.
